tests/rptest/tests/rbac_test.py

```diff
@@ -173,17 +170,14 @@ def test_members_endpoint(self):
         ) == 0, f"Incorrect 'removed' result: {member_update.removed}"
         assert alice in member_update.added, f"Incorrect member added: {member_update.added[0]}"
 
-        def try_get_members(role):
-            try:
-                res = self.superuser_admin.list_role_members(role=role)
-                return True, res
-            except:
-                return False, None
-
         self.logger.debug("And check that we can query the role we created")
-        res = wait_until_result(lambda: try_get_members(self.role_name0),
-                                timeout_sec=5,
-                                backoff_sec=1)
+        res = wait_until_result(
+            lambda:
+            (True, self.superuser_admin.list_role_members(role=self.role_name0)
```

Comment:
question(non-blocking): do we need the `True` here or would this work without it as well? Do we just need it for `wait_until_result` to forward the result?

Reply:
Nice, I hadn't noticed that in the `wait_until_result` doc string. I thought the tuple-ness was required.